requirements_base.txt

```diff
@@ -17,7 +17,7 @@ cffi==1.14.0              # via cryptography
 chardet==3.0.4            # via requests
 click==7.1.2              # via flask, pip-tools
 configargparse==1.2.3     # via locust
-coverage==5.1             # via -r requirements.in
+coverage==6.0b1             # via -r requirements.in
```

Comment:
This feels super nit-picky, but I also feel like it could potentially cause problems later: is there a reason to use the pre-release version over the official current version (5.5)?

Reply:
Yes, I was thinking the same thing, but the code safety check was throwing the error because the version was less than the pre-release (6.0b1). 
`-> coverage, installed 5.1, affected <6.0b1, id 41002`

Reply:
Oh, yeah. Sorry I'd missed that part of it.